docs/0.2.x/settings/services/nightscout.md

```diff
@@ -7,12 +7,11 @@
 
 ## Nightscout Introduction
 
-[Nightscout](http://nightscout.info) (NS) is an open-source project that allows real-time access to CGM data via a personal website, smartwatch viewers, or apps and widgets available for smartphones. Setting up a *Nightscout* web app is recommended to visualize your *Trio* closed loop. 
+[Nightscout](https://nightscout.github.io/) (NS) is an open-source project that allows real-time access to CGM data via a personal website, smartwatch viewers, or apps and widgets available for smartphones. Setting up a *Nightscout* web app is recommended to visualize your *Trio* closed loop. 
 
 *Nightscout* allows a user to upload CGM data from various sources to an online database and cloud computing service. The information is then processed and displayed visually as a graph. Plugins also allow more information to be shown about *Trio*. 
 
-Even if you don't choose to share your *Nightscout* site
-with another person, it will be helpful for you to visualize what the loop is doing and what it's been doing, plus generate helpful reports for understanding your data.
+Even if you don't choose to share your *Nightscout* site with another person, it will be helpful for you to visualize what the loop is doing and what it's been doing, plus generate helpful reports for understanding your data.
```

Comment:
Perhaps say “the app” instead of “the loop”.

Reply:
✔️ Feedback addressed  in commit 93c59e9